### PR TITLE
added Verlet integration

### PR DESCRIPTION
**Pull request by Ved-Mahajan**
This is to add verlet integration
# I have done:
- [x] updated `TheAlgorithms.jl` file
- [x] added the code file
- [x] added tests in `test/math.jl`
- [x] `]test` shows all test passed. 

I will close the other PR because of the mess and add each file properly in seperate PRs

### CODE REVIEW

File: src/TheAlgorithms.jl [hunk 1]
```diff
@@ -141,7 +141,8 @@ include("math/perfect_square.jl")
 include("math/sir_model.jl")
 include("math/sum_of_arithmetic_series.jl")
 include("math/sum_of_geometric_progression.jl")
-include("math/sieve_of_eratosthenes.jl")
+include("math/verlet.jl")
```
Comment (ReneganRonin) [inline src/TheAlgorithms.jl:144]: it seems deleting this `include` has caused the codecov issues and failed tests. pls readd them

File: src/TheAlgorithms.jl [hunk 2]
```diff
@@ -67,7 +67,7 @@ export sum_gp
 export surfarea_cube
 export surfarea_sphere
 export trapazoidal_area
-export eratosthenes
+export verlet_integration
```
Comment (ReneganRonin) [inline src/TheAlgorithms.jl:70]: this too has caused the code cove issue because of the deleted export, thus failed tests

File: test/math.jl [hunk 3]
```diff
@@ -232,11 +232,11 @@
...
+  end 
+  
+   @testset "Math: Verlet scheme" begin
+	   @test verlet_integration(x->-x,[0.0,0.1],(1,10))[end][end] == 9.999	
+   end
+
 
-    @testset "Math: Sieve of Eratosthenes" begin
-        @test eratosthenes(20) ==   [2,3,5,7,11,13,17,19]
-        @test eratosthenes(2) == [2]
```
Comment (ReneganRonin) [inline test/math.jl:239]: same here as well. readd this block so tests will pass.

**Review by ReneganRonin — CHANGES_REQUESTED** [review]

**Comment by Azzaare** [comment]
@ReneganRonin Thanks, because of no guidelines and my delay to review PRs, I think things are a bit messy.

We should merge this PR and solve the coverage later! Thanks.

**Comment by Ved-Mahajan** [comment]
Thanks to @ReneganRonin this PR seems good now :smile:

**Review by Azzaare — APPROVED** [review]

**Pull request merged**